apps/examples/src/hooks/usePerformance.ts

```diff
@@ -0,0 +1,23 @@
+import { TLUiEventSource, TLUiOverrides, debugFlags, measureCbDuration, useValue } from 'tldraw'
+
+export function usePerformance(): TLUiOverrides {
+	const measurePerformance = useValue(
+		'measurePerformance',
+		() => debugFlags.measurePerformance.get(),
```

Comment:
memory, context (how mayn shapes are currently present), selecting which ones we want to use
console.debug
make it easier to parse FPS - prbly just floor it